unit_test/batched/Test_Batched_BatchedGemm.hpp

```diff
@@ -11,19 +11,20 @@ using namespace KokkosBatched;
 namespace Test {
 template <typename DeviceType, typename ViewType, typename ScalarType,
           typename ParamTagType>
-void impl_test_batched_gemm(const int N, const int matAdim1, const int matAdim2,
-                            const int matBdim1, const int matBdim2,
-                            const int matCdim1, const int matCdim2) {
+void impl_test_batched_gemm_handle(BatchedGemmHandle* batchedGemmHandle,
```

Comment:
I confused this one with testing the handle itself only. Can we rename this function to, maybe, ```impl_test_batched_gemm_handling```?

Reply:
This is just a wrapper that accepts the new handle. I will change it to `impl_test_batched_gemm_with_handle`.

Reply:
```impl_test_batched_gemm_with_handle``` is better than ```impl_test_batched_gemm_handle```. Just to avoid confusion for users when they first look at the code.

Reply:
Agreed.